x-pack/dockerlogbeat/magefile.go

```diff
@@ -128,14 +121,14 @@ func createContainer(ctx context.Context, cli *client.Client) error {
 	return nil
 }
 
-// Build builds docker rootfs container root
+// BuildContainer builds docker rootfs container root
 // There's a somewhat complicated process for this:
 // * Create a container to build the plugin itself
 // * Copy that to a bare-bones container that will become the runc container used by docker
 // * Export that container
 // * Unpack the tar from the exported container
 // * send this to the plugin create API endpoint
-func Build(ctx context.Context) error {
+func BuildContainer(ctx context.Context) error {
```

Comment:
👍 I was struggling with the old name since it had a different meaning than every other Beat.